database/migrations/2019_12_22_015115_create_short_urls_table.php

```diff
@@ -13,7 +13,7 @@ class CreateShortUrlsTable extends Migration
      */
     public function up()
     {
-        Schema::create('short_urls', function (Blueprint $table) {
+        Schema::create(config('short-url.database.urls_table', 'short_urls'), function (Blueprint $table) {
```

Comment:
I think we can probably remove the default values from the `config` helper used in the migrations because it'll always be set?